web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -864,22 +879,8 @@
                     url: urlGetFeature
                   })
                       .done(function(response) {
-                        // TODO: Check WFS exception
                         vectorSource.addFeatures(vectorFormat.
                             readFeatures(response));
-
-                        var extent = ol.extent.createEmpty();
```

Comment:
Removing this code because it is useless. This function is triggered after a zoom (smartly, I think, to detect if we really need to refresh it).

So, zooming at the end is... useless and may be even weird.